components/widgets/ui/src/main/resources/PhenoTips/Widgets.xml

```diff
@@ -4254,7 +4254,7 @@ document.observe("xwiki:dom:loaded", function() {
 
   widgets.FuzzyDatePickerDropdown = Class.create({
     initialize : function(options) {
-      this.span     = new Element('span');
+      this.span     = new Element('span', {"tabIndex":"-1"});
```

Comment:
I doubt this is valid, the `tabindex` attribute doesn't apply to `span` elements.